src/atmega328p/hal/port.rs

```diff
@@ -170,4 +176,11 @@ impl Pin {
             self.toggle();
         }
     }
+
+    /// Change pin mode to output by changing the DDR bit of that pin to 1.
```

Comment:
Inner working should not be part of top comments, user does not care which register you change.